package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@volusion/element-cli",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
I know is not ideal, but we have to update the index.ts file to reflect the new version. Or if you want, you can leave it as 3.0.3, we can merge and release with another PR for the CLI coming soon.

Reply:
pushed up a commit, thanks!